test/unit/orchestrationclient-spec.ts

```diff
@@ -218,74 +186,109 @@ describe("Orchestration Client", () => {
 
     describe("getStatusAll()", () => {
         beforeEach(async () => {
-            sinon.stub(WebhookClient.prototype, "get");
-            this.getStub = (WebhookClient.prototype.get as sinon.SinonStub);
+            nock.cleanAll();
         });
 
         afterEach(async () => {
-            this.getStub.restore();
+            nock.cleanAll();
         });
 
         it("calls expected webhook", async () => {
-            const context = defaultContext;
-            const webhookUrl = new URL(context.bindings.starter.managementUrls.statusQueryGetUri
-                .replace(idPlaceholder, ""));
+            const client = new DurableOrchestrationClient(defaultClientInputData);
 
-            const client = new DurableOrchestrationClient(context);
-            const expectedStatuses: DurableOrchestrationStatus[] = [ ];
-            this.getStub.resolves({ status: 202, body: expectedStatuses});
+            const expectedWebhookUrl = new url.URL(defaultClientInputData.managementUrls.statusQueryGetUri
+                .replace(TestConstants.idPlaceholder, ""));
+
+            const expectedStatuses: DurableOrchestrationStatus[] = [
+                new DurableOrchestrationStatus(
+                defaultOrchestrationName,
+                defaultInstanceId,
+                null,
+                null,
+                null,
+                null,
+                OrchestrationRuntimeStatus.Pending,
+                null,
+                null),
```

Comment:
These parameters look like they need to be indented another level.

Reply:
That they do. Thanks for the catch.